src/main/java/com/zaichick/user/entrypoint/CreateUserVerify.java

```diff
@@ -0,0 +1,16 @@
+package com.zaichick.user.entrypoint;
```

Comment:
leave space after package
also move to "service" package "com.zaichick.user.validator"